include/dt-bindings/pinctrl/cc13xx_cc26xx-pinctrl.h

```diff
@@ -0,0 +1,159 @@
+#define IOC_INT_M               0x00070000  // Int config mask
+
+//*****************************************************************************
+//
+// Defines that be used to set pull on an IO
+//
+//*****************************************************************************
+#define IOC_NO_IOPULL           0x00006000  // No IO pull
+#define IOC_IOPULL_UP           0x00004000  // Pull up
+#define IOC_IOPULL_DOWN         0x00002000  // Pull down
+#define IOC_IOPULL_M            0x00006000  // Pull config mask
+#define IOC_IOPULL_M            0x00006000
+
+//*****************************************************************************
+//
+// Defines that can be used to select the drive strength of an IO
+//
+//*****************************************************************************
+#define IOC_CURRENT_2MA         0x00000000  // 2mA drive strength
+#define IOC_CURRENT_4MA         0x00000400  // 4mA drive strength
+#define IOC_CURRENT_8MA         0x00000800  // 4 or 8mA drive strength
+#define IOC_STRENGTH_AUTO       0x00000000  // Automatic Drive Strength
+                                            // (2/4/8 mA @ VVDS)
+#define IOC_STRENGTH_MAX        0x00000300  // Maximum Drive Strength
+                                            // (2/4/8 mA @ 1.8V)
+#define IOC_STRENGTH_MED        0x00000200  // Medium Drive Strength
+                                            // (2/4/8 mA @ 2.5V)
+#define IOC_STRENGTH_MIN        0x00000100  // Minimum Drive Strength
+                                            // (2/4/8 mA @ 3.3V)
+
+//*****************************************************************************
+//
+// Defines for standard IO setup
+//
+//*****************************************************************************
+#define IOC_STD_INPUT           (IOC_CURRENT_2MA | IOC_STRENGTH_AUTO |      \
+                                 IOC_NO_IOPULL | IOC_SLEW_DISABLE |         \
+                                 IOC_HYST_DISABLE | IOC_NO_EDGE |           \
+                                 IOC_INT_DISABLE | IOC_IOMODE_NORMAL |      \
+                                 IOC_NO_WAKE_UP | IOC_INPUT_ENABLE )
+#define IOC_STD_OUTPUT          (IOC_CURRENT_2MA | IOC_STRENGTH_AUTO |      \
+                                 IOC_NO_IOPULL | IOC_SLEW_DISABLE |         \
+                                 IOC_HYST_DISABLE | IOC_NO_EDGE |           \
+                                 IOC_INT_DISABLE | IOC_IOMODE_NORMAL |      \
+                                 IOC_NO_WAKE_UP | IOC_INPUT_DISABLE )
+ #define IOC_STD_I2C            (IOC_CURRENT_2MA | IOC_STRENGTH_AUTO |              \
+                                 IOC_IOPULL_UP | IOC_SLEW_DISABLE |                 \
+                                 IOC_HYST_DISABLE | IOC_NO_EDGE |                   \
+                                 IOC_INT_DISABLE | IOC_IOMODE_OPEN_DRAIN_NORMAL |   \
+                                 IOC_NO_WAKE_UP | IOC_INPUT_ENABLE )                 
```

Comment:
standard properties need to be used. For example, `bias-pull-up`. This is encoding pin properties (bias, pull-up, slew-rate, etc.) in a custom way. Ref. https://docs.zephyrproject.org/latest/guides/pinctrl/index.html#pin-control-drivers

Reply:
implemented standard properties